Cargo.toml

```diff
@@ -13,11 +13,11 @@ version = "0.28.0"
 doctest = true
 
 [dependencies]
+arboard = { version = "3.3.0", optional = true, default-features = false, features = ["wayland-data-control"] }
```

Comment:
Can we include this wayland feature only if it's not windows and not macos? At least that will cut the dependencies for some users.

Reply:
I made sure that is already the case. Take a look at https://github.com/1Password/arboard/blob/master/Cargo.toml.

I can do it explicitly in our Cargo.toml, too, if you want.

Reply:
ok, it looks like it's already that way in arboard. Thanks.